eng/pipelines/templates/VerifyCoverageReport.yml

```diff
@@ -0,0 +1,36 @@
+parameters:
+  - name: isDeltaBuild
+    type: string
+
+steps:
+  - task: DownloadBuildArtifacts@0
+    displayName: Download code coverage reports
+    inputs:
+      artifactName: CodeCoverageResults
+      downloadPath: $(System.DefaultWorkingDirectory)
+    condition: ne(variables['SkipCodeCoverage'], 'True')
```

Comment:
Instead of checking this on each of the tasks, should we just check this when importing the template and that way we can just clean this here?

Reply:
I couldn't get it working for some reason. I'll try again later.